src/g2d/pango/basetextoverlay.c

```diff
@@ -3011,9 +3011,9 @@ gst_imx_g2d_base_text_overlay_video_chain (GstPad * pad, GstObject * parent,
 
 missing_timestamp:
   {
-    GST_WARNING_OBJECT (overlay, "buffer without timestamp, discarding");
-    gst_buffer_unref (buffer);
-    return GST_FLOW_OK;
+    GST_WARNING_OBJECT (overlay, "buffer without timestamp, no overlay");
```

Comment:
so this behaves the same as silent as you mentioned, it doesn't display any text?

if you look at line 2815, this seems to be the part that renders the text set directly on the element instead of subtitles, and it doesn't appear to rely on any timestamp code, could we instead jump into that part and render the text, hopefully solving all of our problems?